### DISCUSSION TITLE
Lenge house

### DISCUSSION THREAD

**Discussion by Helderferr** [naverz/zepeto-studio-kor#668] [4. General]
Hello community 

My name is Hélder and im trying to build a world in unity.
im trying to do something like a preview of the clothes that i show in my world you understand better in the photos i show next
can anyone guide me and teach me how to do it ?? 

![IMG_1509](https://user-images.githubusercontent.com/104442354/165320157-6f812909-ffe2-4ebf-8c0e-f4ad545b2fd1.PNG)
![IMG_1511](https://user-images.githubusercontent.com/104442354/165320164-a216f79d-d7dc-4bd4-9f74-22b1f0289102.PNG)
![IMG_1510](https://user-images.githubusercontent.com/104442354/165320165-da329ea5-a4f9-4bb8-9f5c-413141b1a3ad.PNG)

thanks in advance

**Comment by x4cf** [comment]
보여주신 장면의 월드는 빌드잇으로 제작된 월드로 보입니다 :)

**Reply by Helderferr** [reply]
but what i want to do in my world is the clothes preview like this 
![ç](https://user-images.githubusercontent.com/104442354/165473162-78be05ce-1b13-48a6-96f0-972ab772c623.PNG)
![IMG_1510](https://user-images.githubusercontent.com/104442354/165473259-9d2cfca4-ee20-43a4-bea7-b40281d7e45c.PNG)
and then as you can see it gives you the option to buy the outfit
